org.eclipse.jdt.ui.tests.refactoring/resources/ExtractTemp/canExtract/A_test46_in.java

```diff
@@ -1,8 +1,13 @@
 package p;
+
 class A {
-	int f(){
-		f();
-		int u= f();
-		return 0;
+	public static long fibonacci(long number) {
```

Comment:
I modified this test case earlier because the original code would cause StackOverflowError at runtime. I restored the code of the input test and just made some changes to the output.

Reply:
The test code is not run, just compiled.  Don't worry about it being bad code.  The cleanups and compiler still have to handle bad code.

Reply:
Got it.